index.js

```diff
@@ -0,0 +1,22 @@
+const menuOpen = document.querySelector(".bi-list");
+const menuClose = document.querySelector(".close_hamb");
+const containerNav = document.querySelector(".container-nav");
+const navList = document.querySelector(".nav-list");
+const about = document.querySelectorAll(".mobile-nav-links");
+
+menuOpen.addEventListener("click", () => {
+  containerNav.style.display = "flex";
+  navList.style.display = "none";
+});
+
+menuOpen.addEventListener("click", () => {
+  containerNav.style.display = "flex";
+  navList.style.display = "none";
+});
```

Comment:
When I click your close button, the page refreshes. I think the reason for that is that you applied an event listener to the same element twice. Please fix this as it is not the intended behavior with a mobile menu, you may want to take a look at [this resource](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault) as well.